### PR TITLE
Exclude symbols from publishing to symbol server 

### PR DESCRIPTION
**Pull request by epananth**
### To double check:

* [ ] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/core-eng/tree/master/Documentation/Validation

### CODE REVIEW

File: Documentation/CorePackages/Publishing.md [hunk 1]
```diff
@@ -575,3 +575,16 @@ The aka.ms links are generated using the `BuildQuality` parameter that is passed
...
 
 The build qualities that are supported are daily, signed, validated, preview, and ga. All official daily builds that publish using V3 should use the `daily` build quality. Signed and validated builds are generated by the staging process of the release process. Preview and GA links are generated at release time, on release day. All builds that have preview in the release version will be of the `preview` quality. All other builds will be marked as `GA`. GA builds do not append a build quality to the links.
+
+### Can we exclude symbols from publishing to symbols server?
+Yes. 
+
+Create a file eng/SymbolPublishingExclusionsFile.txt in your repo, add the file name that has to be excluded from symbol publishing in SymbolPublishingExclusionsFile.txt.
+
+Eg: 
+  tools/x86_arm/mscordaccore.dll
```
Comment (riarenas) [inline Documentation/CorePackages/Publishing.md:585]: is this from the root of the repo? or from somewhere else?

File: Documentation/CorePackages/Publishing.md [hunk 2]
```diff
@@ -575,3 +575,16 @@ The aka.ms links are generated using the `BuildQuality` parameter that is passed
...
+  tools/x86_arm/mscordbi.dll
+  tools/x64_arm64/mscordaccore.dll
+  tools/x64_arm64/mscordbi.dll 
+
+During the build, arcade will pick up SymbolPublishingExclusionsFile.txt and exclude the symbols mentioned in it.
```
Comment (riarenas) [inline Documentation/CorePackages/Publishing.md:590]: Maybe it should say during publishing? as this will happen in the publishing pipeline.

**Review by riarenas — APPROVED** [review]

**Comment by epananth** [comment]
The reason this fixed failed is cos arcade needed an arcade update. So only the post build thing happened and the first part where we upload the file to the container did not happen. So I am going to recreate this PR and I am running another test in runtime to prove this.

File: Documentation/CorePackages/Publishing.md (hunk 1 above)
Comment (epananth) [reply Documentation/CorePackages/Publishing.md:585]: its from the symbols.nupkg.. Not the whole path from root though..

**Pull request merged**